data-sets-api-server/src/main/java/org/zowe/unix/files/services/UnixFilesService.java

```diff
@@ -15,9 +15,11 @@
 public interface UnixFilesService {
     UnixDirectoryAttributesWithChildren listUnixDirectory(String path);
     
-    UnixFileContentWithETag getUnixFileContentWithETag(String path);
+    UnixFileContentWithETag getUnixFileContentWithETag(String path, boolean convert);
     
     String putUnixFileContent(String path, UnixFileContentWithETag content, boolean convert);
     
+    boolean getShouldUnixFileTagConvert(String path);
```

Comment:
I'd consider dropping the 'get' from this to make it shouldUnixFileTagConvert(), that's more similar to the java boolean method convention is isEquals(), hasChildren() etc

Reply:
Have renamed toshouldUnixFileConvert